{"author":{"id":"e970470018827b6b178556f877e6452433eeadb8d713d3499b58f62b3765b664"},"ops":[{"type":1,"author":{"id":"e970470018827b6b178556f877e6452433eeadb8d713d3499b58f62b3765b664"},"timestamp":1694688706,"metadata":{"github-id":"I_kwDODbKJfs5xB2mo","github-url":"https://github.com/vmiklos/plees-tracker/issues/414","origin":"github"},"nonce":"Q+wd+ySDMDgGKNpaPJNBwdCRvX8=","title":"ArrowOS: export to Nextcloud failed when plees-tracker.csv file already exists","message":"**Reproducer steps**\n1. export to file from plees-tracker and use default filename\n2. export a second time, again using default filename, this time while a file with that name already exsists.\n3. Third step\n\n**Actual result**\n`Export failed: java.io.FileNotFoundException: Failed t...` and exported file has 0 B.\n\n**Expected behavior**\nAsk user to either overwrite existing file or rename the new export filename.\n\nLogcat sent via email.","files":null}]}